include/vctr/Containers/VctrBase.h

```diff
@@ -366,6 +366,24 @@ class VctrBase : private Config,
     /** Subtracts a constant from this in place */
     void operator-= (value_type c);
 
+    //==============================================================================
+    // Math reduction operations
+    //==============================================================================
+    /** Returns the minimal value of all elements */
+    ElementType min() const requires std::totally_ordered<ElementType>;
```

Comment:
Interesting. How do I make sure that a home-made ElementType is totally_ordered, i.e., [that the comparison operators ==,!=,<,>,<=,>= on a type yield results consistent with a [strict total order](https://en.wikipedia.org/wiki/Total_order#Strict_and_non-strict_total_orders) on the type](https://en.cppreference.com/w/cpp/concepts/totally_ordered)?

Reply:
I'd say you need to implement these operators for your custom type in a way that they fulfil this requirement 🤷‍♂️